.mci.yml

```diff
@@ -145,15 +147,19 @@ cxx_driver_variables:
           git clone https://github.com/10gen/mongo-orchestration.git
           cd mongo-orchestration
           python setup.py install
+          cd ..
           echo "Starting Mongo Orchestration..."
           echo "{ \"releases\": { \"default\": \"c:\\\\mongodb\\\\bin\" }, \"last_updated\": \"2014-08-29 20:57:00.000000\" }" > orchestration.config
           mongo-orchestration -f orchestration.config -e default -s wsgiref start
-          cd ..
       linux: &mongo_orchestration_linux
-        ## mongo orchestration is included on linux, no installation needed
+        ## we are going to manually install mongo-orchestration for now on Linux until it is stable enough that we don't always to install from HEAD
         start_mongo_orchestration: |
           echo "Starting Mongo Orchestration..."
           trap 'set +o errexit; mongo-orchestration stop;' EXIT
+          git clone https://github.com/10gen/mongo-orchestration.git
```

Comment:
Echo an additional line here for the installation of mongo orchestration, as above